### ISSUE TITLE
deprecated "paper-card-background-color" 

### ISSUE THREAD

**Issue by porebas** [kalkih/mini-media-player#359]
The theme variable paper-card-background-color is officially removed. It's recommended to use ha-card-background or card-background-color as a replacement.

**Comment by DavidFW1960** [comment]
Yeah and updated themes are removing that variable now as well so I had a screen of white cards because of this

**Comment by FaintGhost** [comment]
same here.

**Comment by kalkih** [comment]
The card is already using `ha-card-background` primarily, it does however fallback to `paper-card-background-color` if `ha-card-background` isn't set, and then white as last resort.

https://github.com/kalkih/mini-media-player/blob/06424f033d30ddf9a8bae655e07bb6b71c80ce22/src/style.js#L108

Do you have link or something to the source of this deprecated / breaking change?

**Comment by DavidFW1960** [comment]
It's a breaking change in Frontend Themes listed here https://www.home-assistant.io/blog/2020/07/22/release-113/ (It's the last breaking change)
Try using the Google Dark Theme from here https://github.com/JuanMTech/google_dark_theme
Juan removed 
```
paper-card-background-color: var(--card-background-color)
```
And the cards all go white. There is no --ha-card-background in that theme and --paper-card-background-color has been removed. I edited the js file for the card to change --paper-card-background-color to --card-background-color and its fixed.
